File: atcoder.jp/contests/tenka1-2019-beginner/tasks/tenka1_2019_a/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const main = () => {
  const [A, B, C] = input().split(" ").map(Number);

  const ans = (A < C && C < B) || (B < C && C < A);
  console.log(ans ? "Yes" : "No");
};

main();

File: atcoder.jp/contests/tenka1-2019-beginner/tasks/tenka1_2019_a/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int A, B, C;
  cin >> A >> B >> C;

  bool ans = (A < C && C < B) || (B < C && C < A);
  cout << (ans ? "Yes" : "No") << endl;
}
